notebooks/data_selection.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.        dcheck_flatfield = vstack([dcheck_flatfield, read_table(file, "/flatfield/table")])
</pre><p>This vstacking in each loop iteration is very inefficient. It is much better to use a list and call vstack once at the end:</p><p><code></code><code></p><p>dcheck_cosmics = []</p><p>for f in files:</p><p>    dcheck_cosmics.append(read_table(...))</p><p> </p><p>dcheck_cosmics = vstack(dcheck_cosmics)</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>